ckanext/canada/tables/contracts.yaml

```diff
@@ -1711,8 +1702,12 @@ iii. Quand un contrat porte sur plus d’un article économique, il est recomman
     description:
       en: It is recommended for this field to indicate whether the procurement is subject one or more of the applicable Socio-economic indicator(s)
       fr: Il est recommandé d’indiquer dans cette zone si l’approvisionnement fait l’objet d’au moins un des indicateurs socioéconomiques applicables.
-    datastore_type: text
+    validation:
+      en: If the value NA (None) is entered, then no other value can be entered in this field.
+      fr:
+    datastore_type: _text
     excel_full_text_choices: false
+    excel_error_formula: 'AND(ISNUMBER(SEARCH("NA",{cell})),{cell}<>"NA")'
```

Comment:
this makes no sense as a multivalued field, just remove the error_formula and make the datastore_type text again.

Reply:
Yeah I noticed that, so basically we'll just dismiss their validation change request for this field.